packages/subgraph/src/mappings/order.ts

```diff
@@ -26,7 +34,25 @@ export function handleDCAOrderInitialized(event: Initialized): void {
   order.sellToken = createOrReturnTokenEntity(orderContract.sellToken()).id;
   order.buyToken = createOrReturnTokenEntity(orderContract.buyToken()).id;
   order.receiver = orderContract.receiver();
+
+  let protocolFee: BigInt = BigInt.fromI32(0);
+  let orderFactoryAddress: Address = (
+    event.transaction.to !== null
+      ? event.transaction.to
+      : Address.fromString("0x0")
+  )!;
```

Comment:
Minor, non-blocking, this suggestion should yield the same result:

```suggestion
  let orderFactoryAddress: Address = (
    event.transaction.to ?? Address.fromString("0x0")
  )!;
```

More info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
Since this is AssemblyScript it doesn't support the operator you are suggesting. But nice suggestion anyway :)